src/vs/workbench/parts/preferences/browser/settingsEditor2.ts

```diff
@@ -126,7 +138,8 @@ export class SettingsEditor2 extends BaseEditor {
 		this.viewState = { settingsTarget: ConfigurationTarget.USER };
 		this.delayRefreshOnLayout = new Delayer(100);
 
-		this.settingUpdateDelayer = new Delayer<void>(200);
+		this.settingFastUpdateDelayer = new Delayer<void>(SettingsEditor2.SETTING_UPDATE_FAST_DEBOUNCE);
```

Comment:
Possibly should just invoke callbacks without a debounce? Only makes sense for text inputs, but for future-proofing could be useful to keep.

Reply:
I think we should keep the fast debounce. Lots of stuff happens when configuration changes, whether it was for anything important or not, and we should at least sort of avoid updating the config every time if someone clicks a checkbox as fast as they can.